integration/tests/cook/test_basic.py

```diff
@@ -870,7 +898,7 @@ def query():
             # (ensure it still works when there are no live jobs)
             util.kill_groups(self.cook_url, [group_uuid])
 
-    def test_group_change_killed_retries(self):
+    def _help_test_group_change_killed_retries(self, failed_only):
```

Comment:
Maybe we could name this something like `group_submit_kill_retry`? Should it be in `util`?